README.md

```diff
@@ -171,7 +197,7 @@ This dbt package is dependent on the following dbt packages. For more informatio
 ```yml
 packages:
     - package: fivetran/salesforce_source
-      version: [">=0.6.0", "<0.7.0"]
+      version: [">=0.7.0", "<0.8.0"]
```

Comment:
Can you also update the suggested version of this package to install.

line 61 in the readme